cmd/kubelet/app/options/options.go

```diff
@@ -550,4 +550,9 @@ Runtime log sanitization may introduce significant computation overhead and ther
 
 	// Graduated experimental flags, kept for backward compatibility
 	fs.BoolVar(&c.KernelMemcgNotification, "experimental-kernel-memcg-notification", c.KernelMemcgNotification, "Use kernelMemcgNotification configuration, this flag will be removed in 1.23.")
+
+	// Memory Manager Flags
+	fs.StringVar(&c.MemoryManagerPolicy, "memory-manager-policy", c.MemoryManagerPolicy, "Memory Manager policy to use. Possible values: 'none', 'static'. Default: 'none'")
```

Comment:
Please see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants which now clarifies how constants must be used.  New flags need to follow the rules for constants and be `None` and `Static`.  If you would like to preserve consistency with other policy flags, you may support both versions (`None`/`none` and `Static`/`static`)

Reply:
Will it be ok to update the comment to include
```
'None/none', 'Static/static'. Default: 'none'
```

Reply:
Comment should describe the value we want users to use for new fields.  I might buy the argument that you should do this for all flags at once, in which case we can decouple that from this PR (but I'd expect htat to be a criteria for beta)

Reply:
done